pkg/certs/cert.go

```diff
@@ -203,8 +204,10 @@ func (s *simpleCertificateIssuer) IssueCertAndKey(ip string, dns ...string) ([]b
 		altNames.DNSNames = append(altNames.DNSNames, dns...)
 	}
 
-	if len(ip) != 0 {
-		altNames.IPs = append(altNames.IPs, net.ParseIP(ip))
+	if len(ips) != 0 {
+		for _, ip := range ips {
+			altNames.IPs = append(altNames.IPs, net.ParseIP(ip))
+		}
```

Comment:
We don't need this judgment if it's an array

```suggestion
for _, ip := range ips {
	altNames.IPs = append(altNames.IPs, net.ParseIP(ip))
}
```

Reply:
yeah